tests/acceptance/features/bootstrap/PasswordPolicyContext.php

```diff
@@ -21,9 +21,11 @@
  */
 
 use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
 use Behat\Behat\Hook\Scope\BeforeScenarioScope;
 use TestHelpers\AppConfigHelper;
 use TestHelpers\SetupHelper;
+use PhpParser\Node\Stmt\Foreach_;
```

Comment:
why is this needed?

Reply:
Oh, we don't need this. I should remove it.